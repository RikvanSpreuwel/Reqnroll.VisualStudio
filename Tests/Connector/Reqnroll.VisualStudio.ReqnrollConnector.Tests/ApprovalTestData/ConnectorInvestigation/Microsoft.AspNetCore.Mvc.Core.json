{
  "TargetFolder": "$(ApprovalTestData)\\Microsoft.AspNetCore.Mvc.Core",
  "TestAssembly": "ReqnrollProject.dll",
  "ConfigFile": null
}
