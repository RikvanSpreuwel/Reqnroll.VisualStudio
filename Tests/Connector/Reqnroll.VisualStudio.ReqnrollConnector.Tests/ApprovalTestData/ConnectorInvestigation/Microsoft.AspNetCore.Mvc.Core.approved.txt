{
  "stepDefinitions": [],
  "sourceFiles": {},
  "typeNames": {},
  "analyticsProperties": {
    "Connector": "<connector>",
    "ImageRuntimeVersion": "v4.0.30319",
    "SFFile": "Reqnroll.xUnit.ReqnrollPlugin.dll",
    "SFFileVersion": "3.9.40.46609",
    "SFProductVersion": "3.9.40+11b6bbec89",
    "SourcePaths": "0",
    "StepDefinitions": "4",
    "TargetFramework": ".NETCoreApp,Version=v6.0",
    "TypeNames": "0"
  }
}